ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -44,17 +63,17 @@ public void addFormTest() throws Exception {
         manageForm.formIdFromUrl();
         manageForm.save();
         headerPage.clickOnHomeIcon();
-        homePage.goToActiveVisitPatient();
```

Comment:
once again, clarify why this was removed

Reply:
This is not needed with in this code, it will only break the automation perhaps it also doesnt add up within the workflow

Reply:
what makes it break the automation? this line is meant to find the last patient with an active visit. we definitely  need this step and i see no need of dropping it as that breaks the workflow

Reply:
Resolved thanks